instrumentation/servlet/servlet-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/dispatcher/RequestDispatcherTracer.java

```diff
@@ -16,6 +16,6 @@ public static RequestDispatcherTracer tracer() {
 
   @Override
   protected String getInstrumentationName() {
-    return "io.opentelemetry.javaagent.servlet";
+    return "io.opentelemetry.javaagent.servlet.request";
```

Comment:
I don't know if I like it but I guess for consistency and to reflect "name of instrumentation library", these should use `.common`?

Reply:
yeah, I wasn't really sure what these should say. `.common` seems to accurately and uniquely identify the instrumentation, though, and that's definitely the most important thing.

Reply:
updated